app/views/developer/show_developer.html.erb

```diff
@@ -0,0 +1,93 @@
+
+<% if flash[:error]%>
+
+  <div class = "container">
+    <div class="alert alert-danger" role="alert">
+      <%= flash[:error]%>
+    </div>
+    
+  </div>
+  
+  
+
+  <% end %>
+
+<% if flash[:notice]%>
+
+  <div class = "container">
+    <div class="alert alert-info" role="alert">
+      <%= flash[:notice]%>
+    </div>
+    
+  </div>
+  
+  
+
+  <% end %>
```

Comment:
This is repetitive code and is being used in other files as well. Kindly refactor this and handle this in the layout